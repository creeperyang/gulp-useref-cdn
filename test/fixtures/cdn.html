<html>
<head>
<!-- build:cdn /css/combined.css -->
<link href="/css/one.css" rel="stylesheet">
<link href="/css/two.css" rel="stylesheet">
<!-- endbuild -->
</head>
<body>
<!-- build:cdn scripts/combined.js -->
<script type="text/javascript" src="scripts/this.js"></script>
<script type="text/javascript" src="scripts/that.js"></script>
<!-- endbuild -->
</body>
</html>