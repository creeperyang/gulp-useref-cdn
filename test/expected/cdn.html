<html>
<head>
<link rel="stylesheet" href="http://7xkuzg.com1.z0.glb.clouddn.com/combined-f9ebd308.css">
</head>
<body>
<script src="http://7xkuzg.com1.z0.glb.clouddn.com/combined-ce2a0c47.js"></script>
</body>
</html>